shared-artifacts-design.md

```diff
@@ -26,12 +26,10 @@ flowchart TD;
 | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- |
-| author  | concept set  | uncreated | created | viewable to all users in Atlas and GWAS | MVP users have 'shared reader' role for specific artifact |
-| user  | concept set   | viewable to all users | copy | viewable to user's group and authors| MVP team role is assigned to the specific artifact, 'shared reader' role removed |
-| author  | cohort def   | uncreated | created | viewable to all users in Atlas and GWAS | MVP users have 'shared reader' role for specific artifact  |
-| user  | cohort def   | viewable to users in the team in Atlas and GWAS | useful to team | viewable to users in the team in Atlas and GWAS | MVP users have 'shared reader' role for specific artifact  |
-| user  | cohort def   | viewable to users in the team in Atlas and GWAS | user copy | viewable to user's group and authors  (Atlas only) | MVP team role is assigned to the specific artifact, 'shared reader' role removed  |
-| user  | cohort def   | viewable to user's group only and authors (Atlas) | user generate | viewable to user's group only and authors (Atlas and GWAS) | MVP team role retained for generated cohort, no 'shared reader' role |
-| author  | cohort def | viewable to user's group only and authors (Atlas and GWAS) | authors decide to share to all MVP users, artifact copied into an author's account  | viewable to all users in Atlas and GWAS | MVP users have 'shared reader' role for specific artifact |
-
-
+| author  | concept set  | uncreated | created | viewable to all users in Atlas | MVP users have 'shared reader' role for specific artifact |
+| user  | concept set   | viewable to all users in Atlas | copy | viewable to user's group and authors| MVP team role is assigned to the specific artifact, 'shared reader' role removed |
+| author  | cohort def   | uncreated | created | viewable to all users in Atlas, can be generated by authors | MVP users have 'shared reader' role for specific artifact  |
+| author  | cohort def   | viewable to all users in Atlas, can be generated by authors | generate | viewable to all users in Atlas and GWAS | MVP users have 'shared reader' role for specific artifact  |
+| user  | cohort def   | viewable to users in the team in Atlas and GWAS | user copy | copy is viewable to user's group and authors  (Atlas only) | MVP team role is assigned to the specific artifact, 'shared reader' role removed  |
+| user  | cohort def   | viewable to user's group only and authors (Atlas) | user generate | copy is viewable to user's group only and authors (Atlas and GWAS) | MVP team role retained for generated cohort, no 'shared reader' role |
+| author  | cohort def | copy is viewable to user's group only and authors (Atlas and GWAS) | authors decide to share to all MVP users, artifact copied into an author's account  | copy of copy is viewable to all users in Atlas | MVP users have 'shared reader' role for specific artifact |
```

Comment:
the webide diff is better:
![Screenshot 2024-03-15 at 19 37 07](https://github.com/vinci-ohdsi/WebAPI-2-15-dev/assets/2900303/165ae1a9-d9f3-43bb-a298-4963a8be9bf3)